lambda-notes/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Please remove this file from your directory and push again. This causes lots of problems down the line.